owl/EASE-OBJ.owl

```diff
@@ -79,6 +90,116 @@
     
 
 
+    <!-- 
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Data properties
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#hasDepth -->
+
+    <owl:DatatypeProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#hasDepth">
+        <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasDataValue"/>
+        <rdfs:domain rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Entity"/>
```

Comment:
Entity is too large a domain for hasDepth.

Reply:
same as above, and same for the identical comments below.

Reply:
I'm not moved by that answer. Width, depth, length are here physical dimensions of physical objects. This is not about construing things metaphorically.